pkg/apis/machine/v1beta1/machine_webhook.go

```diff
@@ -232,6 +242,156 @@ func getMachineDefaulterOperation(platformStatus *osconfigv1.PlatformStatus) mac
 	}
 }
 
+// NewValidatingWebhookConfiguration creates empty validation webhook configuration
+func NewValidatingWebhookConfiguration() *admissionregistrationv1.ValidatingWebhookConfiguration {
+	return &admissionregistrationv1.ValidatingWebhookConfiguration{
+		ObjectMeta: v1.ObjectMeta{
+			Name: DefaultWebhookConfigurationName,
+			Annotations: map[string]string{
+				"service.beta.openshift.io/inject-cabundle": "true",
+			},
+		},
+	}
+}
+
+// WithValidatingHooks returns validating webhooks list to populate configuration
+func WithValidatingHooks(namespace string) []admissionregistrationv1.ValidatingWebhook {
+	policy := admissionregistrationv1.Ignore
+	sideEffects := admissionregistrationv1.SideEffectClassNone
```

Comment:
Also can we please keep the comments we have in the yaml on why we are choosing this parameters for the policy and the sideEffects